### PR TITLE
POWR-2120 path alias patterns for locations and updates to location management

### PR DESCRIPTION
**Pull request by joshuami**

### CODE REVIEW

File: web/sites/default/config/core.entity_view_display.node.service_location.default.yml [hunk 1]
```diff
@@ -40,14 +45,33 @@ mode: default
...
+    label: hidden
+    settings:
+      format: default
+      format_custom_false: ''
+      format_custom_true: ''
+    third_party_settings: {  }
+    type: boolean
+    region: content
+  field_floor:
+    type: instagram_embed
```
Comment (kxwang) [inline web/sites/default/config/core.entity_view_display.node.service_location.default.yml]: Is the floor field supposed to be "String"?

File: web/sites/default/config/core.entity_view_display.node.service_location.default.yml [hunk 2]
```diff
@@ -40,14 +45,33 @@ mode: default
 content:
   field_body_content:
     type: text_default
-    weight: 4
+    weight: 6
     region: content
     label: hidden
     settings: {  }
     third_party_settings: {  }
+  field_emergency_closure:
```
Comment (kxwang) [inline web/sites/default/config/core.entity_view_display.node.service_location.default.yml]: Is there other types of closures? Or will that info be in open hours field?

File: web/sites/default/config/core.entity_view_display.node.service_location.default.yml (hunk 1 above)
Comment (joshuami) [reply web/sites/default/config/core.entity_view_display.node.service_location.default.yml]: Good catch! I'll get that fixed.

File: web/sites/default/config/core.entity_view_display.node.service_location.default.yml (hunk 2 above)
Comment (joshuami) [reply web/sites/default/config/core.entity_view_display.node.service_location.default.yml]: We may add additional closure types later. When everything is normal again, I could see us creating a feature that allowed location managers to set the schedule of open hours including holidays to show when that location is listed. For now, other types of closures could be described in text in the open hours field.

File: web/sites/default/config/core.entity_view_display.node.service_location.default.yml (hunk 1 above)
Comment (joshuami) [reply web/sites/default/config/core.entity_view_display.node.service_location.default.yml]: Fixed.

**Pull request merged**